Dockerfiles/Dockerfile

```diff
@@ -39,7 +39,7 @@ COPY main.go main.go
 COPY pkg/ pkg/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o manager main.go
```

Comment:
Following this reasoning, I wonder if we need `GOOS` here too?

Reply:
you are right, we can remove GOOS as well.

Reply:
Removed GOOS too, ptal at the latest code.